src/main/java/pl/put/poznan/buildingInfo/model/Building.java

```diff
@@ -57,17 +57,21 @@ public Float getCube() {
         return (float) (levels.stream().mapToDouble(Location::getCube)).sum();
     }
 
-    public Float getHeating() {
-        // calculate heating demand of all levels
-
-        // return temporary value in order to compile code properly
-        return (float) 0.0;
+    /**
+     * Function that return heating of whole building.
+     *
+     * @return heating of all levels
+     */
+    public Float getHeating()   {
+        return (float) (levels.stream().mapToDouble(Location::getHeating)).sum();
```

Comment:
This looks familiar :D

Reply:
At least it's all consistent and uniform ;D